pkg/addons/registry/static/metadata.yaml

```diff
@@ -12,4 +12,4 @@ images:
         repo: proxy.replicated.com/anonymous/replicated/ec-registry
         tag:
             amd64: 2.8.3-r0@sha256:5b76ebd0a362009e31a05ac487c690f5ece0e11f6c4d9261ca63a3f162b57660
```

Comment:
why wasn't this updated?

Reply:
this doesnt work for `replicated/ec-registry` image because its not multi-arch. we need to re-push it

Reply:
how did it work for the arm image then?

Reply:
tag 2.8.3-r0 is an arm image